🛠😶🦑 restoring @nissimpradhan blockchain package

mondial7 opened the pull request.

Commit:
🛠😶🦑 restoring @nissimpradhan blockchain package

martinzumarraga reviewed (commented):
I'm not sure this singleton is well implemented. Check OrderProxy, which is singleton as well

Commit:
🌊 🛠 refactoring and adding comments for missing features

Commit:
🛠 removing todo

nissimpradhan committed:
integration with orderproy

added getTransactionGroupStatus to blockchain class
changed FactoryChain to ProductionChain
corrected singleton to use static modifier
